migrations/20201123155842_store-daily-stats.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex) {
+    return knex.schema.createTable("daily_stats", (table) => {
+        table.date("date").notNullable();
```

Comment:
should this be unique

Reply:
I marked it as unique below. Originally I added unique on the same line, but I followed how you did it in another file